pangram/Pangram.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+public static class Pangram
+{
+    private const int numLettersInAlphabet = 26;
+
+    public static bool IsPangram(string input)
+    {
+        HashSet<char> usedLetters = new HashSet<char>();
+        foreach (char c in input.ToLower())
+        {
+            if (Char.IsLetter(c))
```

Comment:
Think you'll also want to check that the set doesn't already contain the char (if I'm reading things correctly).

Reply:
> I think if there are multiples of one or more letters then it would return false even if the string included all letters.

There are multiple tests that cover this. The HashSets cant contain duplicates so adding a character more than once won't hurt. https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1?view=netcore-3.1#remarks

Reply:
Gotcha. Didn't realize HashSet enforced unique. Now I'm curious which, if either, method would be faster.

Reply:
Not sure... I just spend the last 2 hours trying to figure out how to benchmark xunit tests in dotnet core and I couldn't figure it out.

Reply:
Ergh.